docs/development/almalinux-os-kitten-10.md

```diff
@@ -2,7 +2,7 @@
 title: 'AlmaLinux OS Kitten 10'
 ---
 
-###### last modified: 2025-02-26
+###### last modified: 2025-03-24
```

Comment:
I think this should be automated rather than putting such things by hand. I remember Vuepress has such a feature. I'll try it.

Reply:
See #595.